test/new-e2e/system-probe/test-json-review/codeowners.go

```diff
@@ -0,0 +1,84 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux
+
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+)
+
+// codeowners is a struct that holds the owners of the code. Not a full implementation, just
+// enough to show owners in KMT tests
+type codeowners struct {
+	owners map[string]string
+}
+
+func loadCodeowners(path string) (*codeowners, error) {
+	f, err := os.Open(path)
+	if err != nil {
+		return nil, fmt.Errorf("open %s: %s", path, err)
+	}
+	defer f.Close()
+
+	return loadCodeownersWithReader(f), nil
+}
+
+func loadCodeownersWithReader(contents io.Reader) *codeowners {
```

Comment:
Is there not a package that supports parsing codeowners?

Reply:
There is https://github.com/hmarr/codeowners but I thought that given our simplified needs for this, it was better to not bring an external dependency into this.